docs/src/syntax-highlighting/index.css

```diff
@@ -24,6 +24,4 @@
   --syntax-entity: var(--color-blue);
   --syntax-parameter: var(--color-blue);
   --syntax-variable: var(--color-blue);
-  --syntax-keyword: var(--color-blue);
-  --syntax-string: var(--color-blue);
```

Comment:
I had added this in a previous PR trying to improve one area where inline highlighting felt odd and broke how everything else looked